### PR TITLE
describe both secret and service_account

### PR DESCRIPTION
**Pull request by honnix**
@TinaRanic PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +171,39 @@ In fact, it is so common that we need to use a "last hour" parameter in jobs tha
 default offset for all the well known (aliased) schedules to +1 period. E.g for an `@hourly`
 schedule, the default offset is `PT1H`, and for a `@daily` schedule the offset is `P1D`
 
+#### `schedule[].secret` **[secret]**
+Secret is used to mount keys stored in [Kubernetes Secrets] into the container.
```
Comment (TinaRanic) [inline README.md]: Im thinking if maybe should stay on a higher level (meaning to abstract away the Kubernetes part)

maybe something like:
A Secret is a "container" for keys needed for the workflow container to run and have access to particular resources.

what do you think?

File: README.md [hunk 2]
```diff
@@ -166,6 +171,39 @@ In fact, it is so common that we need to use a "last hour" parameter in jobs tha
...
+* `.mount_path` **[string]**
+
+  Where the keys of the secret will be appearing in the container
+
+#### `schedule[].service_account` **[email address]**
+The [Service Account] email address belonging to a project in [Google Cloud Platform].
+
+If the workflow intends to use keys of a [Service Account],
+Styx will create both JSON and p12 keys for the specified `service_account`, rotate keys on daily basis,
+and garbage collect unused keys.
```
Comment (TinaRanic) [inline README.md]: @danielnorberg great job!

File: README.md [hunk 3]
```diff
@@ -166,6 +171,39 @@ In fact, it is so common that we need to use a "last hour" parameter in jobs tha
...
+
+It is allowed and perfectly fine to have both `secret` and `service_account` configured for a workflow.
+However users need to make sure `secret.mount_path` doesn't point to `/etc/styx-wf-sa-keys/`; otherwise
+Styx will refuse to trigger the workflow.
+
+In order for Styx to be able to create/delete keys for the `service_account`,
+[Service Account] that used by Styx to communicate with [External services] should be granted with
+`Service Account Key Admin` role for the `service_account`. This step can be done by following
+[Granting Roles to Service Accounts].
+
```
Comment (TinaRanic) [inline README.md]: Crystal clear!

File: README.md (hunk 1 above)
Comment (honnix) [inline README.md]: I think it should be OK to have Kubernetes here for now as we don't actually have other means to store secrets. Plus, this is what we claim in the beginning of the doc: `A data processing job scheduler for Kubernetes`. ;)

**Comment by TinaRanic** [comment]
👍

**Pull request merged**